Configuration/PyReleaseValidation/python/MatrixInjector.py

```diff
@@ -185,24 +187,41 @@ def prepare(self,mReader, directories, mode='init'):
             wmsplit['TTbar_13_ID']=1
             wmsplit['SingleMuPt10FS_ID']=1
             wmsplit['TTbarFS_ID']=1
-            wmsplit['RECODR2_50nsreHLT']=1
-            wmsplit['RECODR2_25nsreHLT']=1
+            wmsplit['RECODR2_50nsreHLT']=5
+            wmsplit['RECODR2_25nsreHLT']=5
             wmsplit['RECODR2_2016reHLT']=5
-            wmsplit['RECODR2_2016reHLT_skimSingleMu']=5
-            wmsplit['RECODR2_2016reHLT_skimDoubleEG']=5
-            wmsplit['RECODR2_2016reHLT_skimMuonEG']=5
-            wmsplit['RECODR2_2016reHLT_skimJetHT']=5
-            wmsplit['RECODR2_2016reHLT_skimMET']=5
-            wmsplit['RECODR2_2016reHLT_skimSinglePh']=5
-            wmsplit['RECODR2_2016reHLT_skimMuOnia']=5
+            wmsplit['RECODR2_50nsreHLT_HIPM']=5
+            wmsplit['RECODR2_25nsreHLT_HIPM']=5
+            wmsplit['RECODR2_2016reHLT_HIPM']=1
+            wmsplit['RECODR2_2016reHLT_skimSingleMu']=1
+            wmsplit['RECODR2_2016reHLT_skimDoubleEG']=1
+            wmsplit['RECODR2_2016reHLT_skimMuonEG']=1
+            wmsplit['RECODR2_2016reHLT_skimJetHT']=1
+            wmsplit['RECODR2_2016reHLT_skimMET']=1
+            wmsplit['RECODR2_2016reHLT_skimSinglePh']=1
+            wmsplit['RECODR2_2016reHLT_skimMuOnia']=1
+            wmsplit['RECODR2_2016reHLT_skimSingleMu_HIPM']=1
+            wmsplit['RECODR2_2016reHLT_skimDoubleEG_HIPM']=1
+            wmsplit['RECODR2_2016reHLT_skimMuonEG_HIPM']=1
+            wmsplit['RECODR2_2016reHLT_skimJetHT_HIPM']=1
+            wmsplit['RECODR2_2016reHLT_skimMET_HIPM']=1
+            wmsplit['RECODR2_2016reHLT_skimSinglePh_HIPM']=1
+            wmsplit['RECODR2_2016reHLT_skimMuOnia_HIPM']=1
             wmsplit['HLTDR2_50ns']=1
             wmsplit['HLTDR2_25ns']=1
             wmsplit['HLTDR2_2016']=1
             wmsplit['Hadronizer']=1
-            wmsplit['DIGIUP15']=5
-            wmsplit['RECOUP15']=5
+            wmsplit['DIGIUP15']=1 
+            wmsplit['RECOUP15']=1 
             wmsplit['RECOAODUP15']=5
             wmsplit['DBLMINIAODMCUP15NODQM']=5
+            wmsplit['DigiFull']=5
+            wmsplit['RecoFull']=5
+            wmsplit['DigiFullPU']=1
+            wmsplit['RecoFullPU']=1
+            wmsplit['RECOHID11']=1
+            wmsplit['DigiFullPU_2023D17PU']=1
```

Comment:
probably these 2023-specific steps can be removed too

Reply:
Thx, done. Please check.